src/content/reference/react/Children.md

```diff
@@ -2,6 +2,10 @@
 title: Children
 ---
 
+<AddedInVersion version="0.10.0" />
+
+[//]: # (Does not show up in changelogs but only appears in v0.10.0 docs based on https://web.archive.org/ research)
```

Comment:
```suggestion
{/*(Does not show up in changelogs but only appears in v0.10.0 docs based on https://web.archive.org/ research)*/}
```

Reply:
Thanks. I just `ctrl + /` in my editor and it added the syntax for what it thought was an mdx comment